lib/ug.js

```diff
@@ -1,19 +1,33 @@
 const { Client } = require('ldapts')
 const log = require('skog')
 
+let ldapClient
+
+async function ldapBind () {
+  log.info('Connecting to UG via LDAP...')
+  ldapClient = new Client({
+    url: process.env.UG_URL
+  })
+  await ldapClient.bind(process.env.UG_USERNAME, process.env.UG_PASSWORD)
+}
+
+async function ldapUnbind () {
+  log.info('Disconnecting to UG via LDAP...')
+  await ldapClient.unbind()
```

Comment:
I believe this should also remove the ldapClient variable/set it to null or something. Since on line 25 we check if it is truthy, which it will be even if it is unbound